react/ComplementName.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import useProduct from 'vtex.product-context/useProduct'
+import { useCssHandles } from 'vtex.css-handles'
+
+const CSS_HANDLES = [
+  'productComplementNameWrapper',
+  'productComplementName',
+] as const
+
+const ComplementName: React.FC = () => {
+  const { handles } = useCssHandles(CSS_HANDLES)
+  const { product } = useProduct()
+
+  const complementName = product?.sku?.complementName
+
+  return (
+    <div className={`${handles.productComplementNameWrapper} t-body`}>
+      <span className={handles.productComplementName}>{complementName}</span>
+    </div>
+  )
+}
+
+export default ComplementName
```

Comment:
I think this should be a custom react block. I think it may be very specific to your use case, and it's easily buildable, but supporting one more block in the future may come at a cost for us. Would you mind releasing it in your own app?

Reply:
This is an very used information of products in our stores, but I understand the possible future cost. This change shouldn't have been pushed, I wold make another PR for this. I will remove and then push again